lib/autoupdate/agent/process.go

```diff
@@ -371,15 +372,19 @@ func (s SystemdService) Enable(ctx context.Context, now bool) error {
 	if err := s.checkSystem(ctx); err != nil {
 		return trace.Wrap(err)
 	}
-	args := []string{"enable", s.ServiceName}
-	if now {
-		args = append(args, "--now")
-	}
-	code := s.systemctl(ctx, slog.LevelInfo, args...)
+	// The --now flag is not supported in systemd versions older than 220,
+	// so perform enable + start commands instead.
+	code := s.systemctl(ctx, slog.LevelInfo, "enable", s.ServiceName)
```

Comment:
I looks like both RHEL7 and CentOS7 have `systemctl enable --now`:

```bash
$ docker run -it --rm centos:7 bash -c "systemctl enable --help | grep now"
     --now            Start or stop unit in addition to enabling or disabling it
```

```bash
$ [root@hugo-autoupdate-test-rhel7 ec2-user]# cat /etc/redhat-release 
Red Hat Enterprise Linux Server release 7.9 (Maipo)

$ [root@hugo-autoupdate-test-rhel7 ec2-user]# systemctl enable --help | grep "now"
     --now            Start or stop unit in addition to enabling or disabling it
```

Do we still want to do this change>

Reply:
`--now` was added in 220: https://www.freedesktop.org/software/systemd/man/latest/systemctl.html#--now

Since we're aiming to support a wide range of distros, and older versions of rhel7/centos7 will not have the backport, it seems safer to stick to the spec and check the version for any APIs that have version requirements.